python/cog/_vendor/vendor.txt

```diff
@@ -0,0 +1,24 @@
+attrs==22.2.0
+fastapi==0.95.1
+    starlette==0.26.1
+    anyio==3.6.2
+    sniffio==1.3.0
+    httpx==0.24.0
+    httpcore==0.17.0
+    h11==0.14.0
+    curio==1.6
+hypothesis==6.72.0
```

Comment:
This is a dev dependency, so I would have thought we don't want to vendor it, right?